client/verta/verta/tracking/_role.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+from .._protos.public.uac import RoleV2_pb2 as _Role
+
+
+class Role(object):
+    """
+    Object representing a Role.
+
+    """
+
+    def __init__(self, conn, msg):
+        self.conn = conn
+        self.msg = msg
+        self.id = msg.id
+        self.org_id = msg.org_id
+        self.name = "{}:{}".format(msg.org_id, msg.name)
+
+    @classmethod
+    def _create(
+        cls, conn, name, org_id, resource_actions
+    ):
+        Message = _Role.SetRoleV2
+        msg = cls._create_msg(name, org_id, resource_actions)
+
+        response = conn.make_proto_request(
+            "POST",
+            "/api/v2/uac-proxy/organization/{}/roles".format(org_id),
```

Comment:
This is nit-picky, especially in this case, but I would love to do away with `.format()` forever.  Now that we have left Python 2 behind, we can use f-strings which are much more readable (IMHO).
`f"/api/v2/uac-proxy/organization/{org.id}/roles",`

Reply:
At least when there is just one variable involved, I agree that an f-string is a bit nicer

```suggestion
            f"/api/v2/uac-proxy/organization/{org_id}/roles",
```